### DISCUSSION TITLE
Acer Chromebook Management Un-Enroll

### DISCUSSION THREAD

**Discussion by Sxri** [3kh0/ext-remover#137] [Fixes JOIN TITANIUM NETWORK TO SHARE PLEASE]
My old school used to use Dell chromebooks. if your school uses dell, i dont know what to tell you.

However, my new school (the one i'm currently at) uses Acer. i found this quick github repository to take over the chromebook if you use acer, and / or unenroll it.

It says its for acer 720, however, I've used it on multiple different acer chromebooks, and they have all so far worked with this method.

### **IF YOU DO THIS, DO IT AT HOME!**

Keep safe, as in this method you will take apart your chromebook, I'm not responsible for you getting in trouble, just keep it secret. dont tell everyone you deleted management

repository: https://github.com/wuseman/Acer_Chromebook_C720

**Comment by synapt0** [comment]
Looking into doing this...

**Reply by 6829ASCS** [reply]
Personally I wouldn't, its way too high risk and really is simply going too far to unblock a chromebook...

**Reply by Sxri** [reply]
> Personally I wouldn't, its way too high risk and really is simply going too far to unblock a chromebook...

Its not really high risk considering its just like any other method but using it manually into the chromebook. You can still enable and disable it by just repeating your steps or putting it back in. The school won't suspect you of it if you don't brag or tell everyone, say its a glitch, bring it home, and put it back in for maybe a week then do it again.

It's a good bypass in the meantime for me atleast.

**Comment by Alex2Mayhem** [comment]
> My old school used to use Dell chromebooks. if your school uses dell, i dont know what to tell you.
> 
> However, my new school (the one i'm currently at) uses Acer. i found this quick github repository to take over the chromebook if you use acer, and / or unenroll it.
> 
> It says its for acer 720, however, I've used it on multiple different acer chromebooks, and they have all so far worked with this method.
> 
> ### **IF YOU DO THIS, DO IT AT HOME!**
> Keep safe, as in this method you will take apart your chromebook, I'm not responsible for you getting in trouble, just keep it secret. dont tell everyone you deleted management
> 
> repository: https://github.com/wuseman/Acer_Chromebook_C720

Taking apart chromebook? No. Bad Idea.

**Reply by 3kh0** [reply]
Some people are desperate

**Reply by ghost** [reply]
ikr

**Reply by Sxri** [reply]
sorta, but if ur desperate then go ahead
